src/test/java/seedu/address/testutil/ClientBuilder.java

```diff
@@ -43,6 +48,11 @@ public ClientBuilder() {
         address = new Address(DEFAULT_ADDRESS);
         country = new Country(DEFAULT_COUNTRY);
         timezone = new Timezone(DEFAULT_TIMEZONE);
+        try {
+            contractExpiryDate = ParserUtil.parseContractExpiryDate(DEFAULT_CONTRACT_EXPIRY_DATE);
+        } catch (ParseException ignored) {
+            // Intentionally left empty, since if contractExpiryDate fails to initialize, other tests will fail as well.
+        }
```

Comment:
Is there a reason that this is instantiated differently from the rest?

Reply:
Yeah its cuz the parsing of a date string always needs to have exception thrown